lib/Parse/ParseStmt.cpp

```diff
@@ -1781,8 +1781,9 @@ Parser::classifyConditionalCompilationExpr(Expr *condition,
           if (!fullCheck) {
             return None;
           }
-          return
-            Context.getModule({ { argumentIdent, UDRE->getLoc() } }) != nullptr;
+
+          auto *M = Context.getModule({ { argumentIdent, UDRE->getLoc() } });
```

Comment:
In retrospect this seems terrifying; loading a module has side effects even when it isn't imported. (Extensions still get hooked up, AnyObject dynamic lookup still finds things, operators still become visible.) All of those things are bugs, but we should probably put a `canImport` operation on ModuleLoader. (That'd be cheaper anyway if you didn't actually end up loading the thing.)